exasol_advanced_analytics_framework/resources/outputs/create_query_loop.sql

```diff
@@ -12,380 +12,224 @@ CREATE OR REPLACE LUA SCRIPT "AAF_RUN_QUERY_HANDLER"(json_str) RETURNS TABLE AS
-        M:new(arg1):raise()
-    else
-        M.create(arg1, ...):raise()
-    end
+function ExaError.error(code, message, parameters, mitigations)
+     ExaError:new(code, message, parameters, mitigations):raise()
 end
 
-return M
+return ExaError
 end
 end
 
 do
 local _ENV = _ENV
-package.preload[ "exasol_script_tools" ] = function( ... ) local arg = _G.arg;
----
--- @module exa_script_tools
---
--- This module contains utilities for Lua Scripts in Exasol
---
-
-local M = {
-}
-
-local exaerror = require("exaerror")
-local json = require("cjson")
-
----
--- Extend `exa-object` with the global functions available in Lua Scripts.
--- See Exasol Scripting: https://docs.exasol.com/db/latest/database_concepts/scripting/db_interaction.htm
--- We provide this function such that implementers of a main function can encapsulate the global objects
--- and properly inject them into modules.
---
--- @param exa exa-object available inside of Lua Scripts
---
--- @return lua table including meta and functions
---
-function M.create_exa_env(exa)
-    local exa_env = {
-        meta = exa.meta,
-        -- We put the global functions into a subtable, such that we can replace the subtable with a mock
-        functions = {
-            pquery = pquery,
-            query = query,
-            error = error
-        }
-    }
-    return exa_env
-end
-
----
--- Parse a given arguments in JSON string format.
---
--- @param json_str input parameters as JSON string
---
--- @return Lua table containing the parameters
---
-function M.parse_arguments(json_str, exa_env)
-    local success, args = pcall(json.decode, json_str)
-    if not success then
-        local error_obj = exaerror:new({
-            code = "E-AAF-1",
-            message = "Arguments could not be converted from JSON object to Lua table: {{raw_json}}",
-            parameters = { raw_json = { value = json_str, description = "raw JSON object" } },
-            mitigations = { "Check syntax of the input string JSON is correct" }
-        })
-        exa_env.functions.error(tostring(error_obj))
-    end
-    return args
-end
-
----
--- Encapsulates the result of a QueryHandler such that it can be returns from a Exasol Lua Script.
---
--- @param result A string containing the result of the QueryHandler
---
--- @return A tuple of a table with a single row and one column and the SQL column definition for it
---
-function M.wrap_result(result)
-    local return_result = { { result } }
-    return return_result, "result_column VARCHAR(2000000)"
-end
-
-return M
-end
-end
-
-do
-local _ENV = _ENV
-package.preload[ "message_expander" ] = function( ... ) local arg = _G.arg;
----
--- This module provides a parser for messages with named parameters and can expand the message using the parameter
+package.preload[ "MessageExpander" ] = function( ... ) local arg = _G.arg;
+--- This class provides a parser for messages with named parameters and can expand the message using the parameter
```

Comment:
```suggestion
--- This class provides a parser for messages with named parameters and can expand the message using the parameters
```

Reply:
Generated file which vendorizes all dependencies. This code is not part of our own codebase.